src/node.cc

```diff
@@ -2177,7 +2177,15 @@ void FatalException(Isolate* isolate,
 
   if (false == caught->BooleanValue()) {
     ReportException(env, error, message);
-    exit(1);
+    if (abort_on_uncaught_exception) {
+#ifdef _WIN32
+      raise(SIGABRT);
+#else
+      abort();
+#endif
```

Comment:
Special-casing abort() here and only here is pretty meh IMO.  At the very least add a comment but personally, I'd add an ABORT() macro to src/util.h and update all calls to abort() in src/ in one fell swoop.